contracts/libs/Controlled.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.4.15;
+
+
+contract Controlled {
+    address public controller;
+    modifier onlyController() {
+        require(isController(msg.sender));
+        _;
+    }
+
+    function Controlled() { controller = msg.sender; }
+
+    function isController(address addr) public returns(bool) { return addr == controller; }
+
+    function transfer(address newController) public onlyController {
```

Comment:
This is fine, but I like `changeController` like [MiniMe uses](https://github.com/Giveth/minime/blob/master/contracts/Controlled.sol). Makes it harder to read as a transfer of tokens or ether.

Reply:
Ah, that makes a lot of sense. Will change!